assets/css/badge.css

```diff
@@ -0,0 +1,99 @@
+
+.badge-avatar-container{
+    display: flex;
+    position: relative;
+    gap: 2rem;
+    flex-wrap: wrap;
+}
+
+.badge-image{
+    position: relative;
+}
+
+.badge-image_a{
+    height: 16vh;
+    min-width: 9vw;
+    border-radius: 50%;
+    position: relative;
+}
+
+.badge-green{
+    height: 2.2vh;
+    width: .5vw;
+    border-radius: 50%;
+    border: .2rem solid var(--status-badge);
+    background-color: var(--green);
+    position: absolute;
+    top: 8.5rem;
```

Comment:
using left  makes badge fixed at that place irrespective of image size

Reply:
Thanks, changes has been done.